feat(modules): use 2/stable track for nats charm in the plan

adglkh opened the pull request:
## Done

The main plan for the controller module bundle now aligns with the
latest nats charm release on channel 2/stable rather than latest/edge, 
which lacks features, e.g.  self-signed-certificate TLS charm support 
and secret support.

## QA

PR workflow would work without a problem

## JIRA / Launchpad bug

Fixes https://warthogs.atlassian.net/browse/AC-3677

## Documentation

Does this impact the team's internal or public documentation? If yes, has the relevant documentation been updated?
A: no, it doesn't.

adglkh committed:
feat(modules): use 2/stable track for nats charm in the plan

The main plan for the controller module bundle now aligns with the
latest nats charm release on track 2/stable rather than latest/edge,
which lacks features, e.g.  self-signed-certificate TLS charm support
and secret support.

jat-canonical commented:
we also need to bump the base version here as I see that the tests are being run from 1.26/edge.

adglkh commented:
> we also need to bump the base version here as I see that the tests are being run from 1.26/edge.

A: oh, I thought we were using 1.26/stable anyway. Let me adjust this too.

adglkh committed:
feat(many): use 1.27/stable as the default deployment channel

Anbox Cloud 1.27 has been released for some time. This change updates
all references to the 1.26/edge or 1.26/stable channels in the plan
to use the latest stable channel, which is 1.27/stable.

adglkh commented:
The public doc was updated too
https://github.com/canonical/anbox-cloud-docs/pull/461

morphis commented:
@adglkh can you please move the node controller change into a separate PR or update the description? Otherwise this looks like an accidental add.

adglkh commented:
> @adglkh can you please move the node controller change into a separate PR or update the description? Otherwise this looks like an accidental add.

A: +1, sure. make sense to me

adglkh commented:
https://github.com/canonical/anbox-cloud-terraform/pull/21

I'll drop the the commit https://github.com/canonical/anbox-cloud-terraform/pull/20/commits/b76f6e75417a9543e3993174ba7c39125ace9f3b
later on and rebase after the above PR is merged

morphis reviewed (approved):
LGTM
